DataverseGen.Core/Generators/Scriban/MultipleFileTemplate.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Scriban;
+using System;
+using System.IO;
+
+namespace DataverseGen.Core.Generators.Scriban
+{
+    public class MultipleFileTemplate
+    {
+        private readonly string _templateName;
+
+        public MultipleFileTemplate(string templateName)
+        {
+            _templateName = templateName;
+            switch (_templateName)
+            {
+                case "Main":
```

Comment:
I think we should store configurations constants in one place ("Main" here, different engine types...)